setup-gcloud/README.md

```diff
@@ -53,4 +53,25 @@ steps:
 
 * `service_account_email`: (Optional) The service account email which will be used for authentication.
 
-* `service_account_key`: The service account key which will be used for authentication. This key should be [created](https://cloud.google.com/iam/docs/creating-managing-service-account-keys), encoded as a [Base64](https://en.wikipedia.org/wiki/Base64) string (eg. `cat my-key.json | base64` on macOS), and stored as a [secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets). 
+* `service_account_key`: The service account key which will be used for authentication. This key should be [created](https://cloud.google.com/iam/docs/creating-managing-service-account-keys), encoded as a [Base64](https://en.wikipedia.org/wiki/Base64) string (eg. `cat my-key.json | base64` on macOS), and stored as a [secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets).
+
+* `service_account_file_name`: (Optional) The name of the file that the service account ADC will be saved into, this is used when you need to reference this file with `GOOGLE_APPLICATION_CREDENTIALS`, if not specify, name will be randomize.
```

Comment:
typo: s/specify/specified

Reply:
Also, please add a note making clear that this will cause a key file to persist on the runner's environment.